js/checkout.js

```diff
@@ -244,7 +244,7 @@ class Blockonomics {
         let left = parseFloat(style.left.replace("px", ""))
 
         setTimeout(() => {ele.style.left = (width + left) + 'px', ele.style.width = '0px'}, 10)
-        setTimeout(() => ele.remove(), 400)
+        setTimeout(() => ele.remove(), 300)
```

Comment:
I added a 100ms extra time so that the animation eases out before we remove it, to make it look less abrupt.